go/libkb/chain_link.go

```diff
@@ -1303,3 +1393,23 @@ func (c ChainLink) AllowStubbing() bool {
 	}
 	return c.unpacked.outerLinkV2.LinkType.AllowStubbing()
 }
+
+// This logic is defined at
```

Comment:
same thing about Go-style comments.  Also, not great to link to a private document in public code.  Can you put in the important parts of that doc here?